### PR TITLE
Update the packages for 5.8 beta1 batch 2

### PR DESCRIPTION
**Pull request by youknowriad**
This updates WP to the latest WP packages. It includes the backports from https://github.com/WordPress/gutenberg/pull/32508

Trac ticket: https://core.trac.wordpress.org/ticket/52991

See the diff in https://github.com/WordPress/gutenberg/pull/32508/files

### CODE REVIEW

File: src/wp-includes/blocks/post-template.php [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+ * This ensures backwards compatibility for any users running the Gutenberg
+ * plugin who have used Query Loop prior to its renaming.
+ *
+ * This can be removed when WordPress 5.9 is released.
+ *
+ * @see https://github.com/WordPress/gutenberg/pull/32514
+ */
+function gutenberg_register_legacy_query_loop_block() {
+	$registry = WP_Block_Type_Registry::get_instance();
+	if ( $registry->is_registered( 'core/query-loop' ) ) {
```
Comment (ntsekouras) [inline src/wp-includes/blocks/post-template.php:123]: Since we are removing now from core we can loose this check.

Comment (youknowriad) [reply src/wp-includes/blocks/post-template.php:123]: This file is automatically brought from Gutenberg, so we can't do this here. 

Another thing is that the function still has the `gutenberg_` prefix (I think this should be removed in a follow-up).

These changes can be done after beta1 though.

File: tests/phpunit/includes/functions.php [hunk 2]
```diff
@@ -318,7 +318,8 @@ function _unhook_block_registration() {
 	remove_action( 'init', 'register_block_core_post_terms' );
 	remove_action( 'init', 'register_block_core_post_title' );
 	remove_action( 'init', 'register_block_core_query' );
-	remove_action( 'init', 'register_block_core_query_loop' );
+	remove_action( 'init', 'register_block_core_post_template' );
+	remove_action( 'init', 'gutenberg_register_legacy_query_loop_block' );
```
Comment (ntsekouras) [inline tests/phpunit/includes/functions.php:322]: Should we remove the `gutenberg` prefix here?

Comment (youknowriad) [reply tests/phpunit/includes/functions.php:322]: https://github.com/WordPress/wordpress-develop/pull/1351#discussion_r647629755

**Review by ntsekouras — COMMENTED** [review]
Tested and looks good without the GB plugin with existing and new Query blocks

**Comment by youknowriad** [comment]
committed.

**Pull request closed without merging**